src/gui/MainWindow.cpp

```diff
@@ -28,6 +28,7 @@ MainWindow::MainWindow(AbstractImageGrabber *imageGrabber, RunMode mode) :
 	mImageAnnotator(new KImageAnnotatorAdapter),
 	mSaveAsAction(new QAction(this)),
 	mUploadAction(new QAction(this)),
+	mCopyToClipboardAction(new QAction(this)),
```

Comment:
We have already a CopyAction which is actually a Copy to Clipboard. I think this one should be called CopyAsDataUri or something like that.